docs/Tutorials/CCE.md

```diff
@@ -232,18 +130,293 @@ The quantities that must be provided by the Bondi-Sachs metric data format are:
+/Cce/News.dat                 Dataset {3995/Inf, 163}
+/Cce/Psi0.dat                 Dataset {3995/Inf, 163}
+/Cce/Psi1.dat                 Dataset {3995/Inf, 163}
+/Cce/Psi2.dat                 Dataset {3995/Inf, 163}
+/Cce/Psi3.dat                 Dataset {3995/Inf, 163}
+/Cce/Psi4.dat                 Dataset {3995/Inf, 163}
+/Cce/Strain.dat               Dataset {3995/Inf, 163}
+/src.tar.gz               Dataset {3750199}
+```
+
+### Raw CCE output
+
+The `Strain` represents the asymptotic transverse-traceless contribution
+to the metric scaled by the Bondi radius (to give the asymptotically leading
+part), the `News` represents the first derivative of the strain, and each
+of the `Psi...` datasets represent the Weyl scalars, each scaled by the
+appropriate factor of the Bondi-Sachs radius to retrieve the asymptotically
+leading contribution.
+
+The `EthInertialRetardedTime` is a diagnostic dataset that represents the
+angular derivative of the inertial retarded time, which determines the
+coordinate transformation that is performed at future null infinity.
+
+If you'd like to visualize the output of a CCE run, we offer a
+[CLI](py/cli.html) that will produce a plot of all of quantities except
+`EthInertialRetardedTime`. To see how to use this CLI, run
+
+```
+spectre plot cce -h
+```
+
+If you'd like to do something more complicated than just make a quick plot,
+you'll have to load in the output data yourself using `h5py` or our
+`spectre.IO.H5` bindings.
+
+\note The CLI can also plot the "old" version of CCE output, described above.
+Pass `--cce-group %Cce` to the CLI. This option is only for backwards
+compatibility with the old CCE output and is not supported for the current
+version of output. This options is deprecated and will be removed in the future.
+
+### Frame fixing
+
+You may notice some odd features in some of the output quantities if you try and
+plot them. This is not a bug in CCE (not that we know of at least). This occurs
+because the data at future null infinity is in the wrong Bondi-Metzner-Sachs
+(BMS) frame. In order to put this data into the correct BMS frame, the
+[SXS Collaboration](https://github.com/sxs-collaboration) offers a python/numba
+code called [scri](https://github.com/moble/scri) to do these transformations.
+See their documentation for how to install/run/plot a waveform at future null
+infinity in the correct BMS frame.
+
+Below is a plot of the imaginary part of the 2,2 component for the strain when
+plotted using the raw output from SpECTRE CCE and BMS frame-fixing with scri.
+You'll notice that there is a non-zero offset, and this component of the strain
+doesn't decay back down to zero during the ringdown. This is because of the
+improper BMS frame that the SpECTRE CCE waveform is in. A supertranslation must
+be applied go transform the waveform into the correct BMS frame. See
+\cite Mitman2024review for a review of BMS transformations and gravitational
+memory.
```

Comment:
```suggestion
memory. The plot below shows the output of SpECTRE CCE as well as the output after using scri to do
the BMS frame transformation (see LINK KEEFE's TUTORIAL for help with how to perform the frame fixing).
```

Reply:
Do we want to include the example code in the `scri` `abd` tutorial for plotting a waveform? Or just link to it?

Reply:
I'm fine just linking to it but have no preference either way :)

Reply:
I already mention that the plot is raw CCE vs scri in the first sentence of this paragraph. I'll add the link to Keefe's tutorial here